.changeset/ten-trees-smoke.md

```diff
@@ -0,0 +1,6 @@
+---
+"@nomicfoundation/hardhat-verify": patch
```

Comment:
The change in `hh-verify` is just a refactor without user impact (except maybe saving a couple of CPU cycles), so we shouldn't include it in the changeset.